src/Utilities/TypeTraits.hpp

```diff
@@ -1360,6 +1360,30 @@ template <typename T>
 using remove_cvref_wrap_t = typename remove_cvref_wrap<T>::type;
 // @}
 
+/// \ingroup TypeTraitsGroup
+/// \brief Permits designating a fundamental type for a container
+///
+/// \details Uses SFINAE to designate `get_fundamental_type<T>::type` as `T`
```

Comment:
Everything in type traits basically uses SFINAE, not something worth mentioning (if it's mentioned on other traits please file an issue and I'll remove it). The effects are interesting and should be documented :)